packages/create-playwright/src/generator.ts

```diff
@@ -139,6 +139,7 @@ export class Generator {
     if (!gitIgnore.includes('node_modules'))
       gitIgnore += 'node_modules/\n';
     gitIgnore += 'test-results/\n';
+    gitIgnore += 'playwright-report\n';
```

Comment:
```suggestion
    gitIgnore += 'playwright-report/\n';
```
nit / personal preference